### ISSUE TITLE
Torch max function update

### ISSUE THREAD

**Issue by gulvarol** [anewell/pose-hg-demo#7]
Hi Alejandro,

I guess you updated the training code to work with the updated 'max' function which doesn't return index anymore. (with this [commit](https://github.com/anewell/pose-hg-train/commit/f61d3c60c7fe1c9a3292ffcf4266e448f87bbad0#diff-705152753a16e107f1f62af619a81490))

But it doesn't seem to be updated in this demo repository. I haven't ran it myself, I just realized looking at the code.

**Comment by anewell** [comment]
Thanks for pointing that out, I think I didn't change it here since the bug only affected the on-the-fly accuracy calculation which doesn't get computed here. I've updated it to keep things consistent.

**Comment by anibali** [comment]
@anewell I think that your update addressing this issue is affecting the results adversely. See https://github.com/anewell/pose-hg-demo/pull/14.
